web/client/plugins/tocitemssettings/defaultSettingsTabs.js

```diff
@@ -42,6 +43,9 @@ const responses = {
 import { StyleSelector } from '../styleeditor/index';
 
 const StyleList = defaultProps({ readOnly: true })(StyleSelector);
+const ConnectedDisplay = connect(
+    createSelector([mapSelector], ({ zoom, projection }) => ({ zoom, projection }))
```

Comment:
can this be a problem if toc is used outside our store? 
i mean what do you think to pass zoom and projection from the TocItemSettingsPlugin.jsx [here](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/plugins/TOCItemsSettings.jsx#L108)?
if not, just ignore this comment